### PR TITLE
Portfolio Mobile Version

### PR DESCRIPTION
**Pull request by kidde60**
### Portfolio Mobile Version.
**On this webpage I:**

- added contact form section.
- added icons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,138 +1,215 @@
...
+            <p>
+              If you have an application you are interested in developing, a feature
+              that you need built or a project that needs coding. I'd love to help
+              with it
+            </p>
+            <form class="form" action="https://formspree.io/f/xlezygkr" method="POST">
+              <input id="name" name="Name" class="text" type="text" placeholder="Text Input" maxlength="30" required>
+                  <input id="email" name="email" class="text" type="email"  placeholder="example@gmail.com" required>
+                  <textarea name="Message" id="message" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                  <button type="submit" class="btn" id="btn">See Project</button>
```
Comment (ishimwezachee) [inline index.html:181]: - [ ] Your Button should have interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi  @kidde60  and @Elijahdre , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Linters are passing 👍 
Projects are running successfully 👍 
Github flow is flowed 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — COMMENTED** [review]
Great job

File: index.html [hunk 2]
```diff
@@ -1,138 +1,215 @@
...
-</body>
-</html>
\ No newline at end of file
+            <h1>CONTACT US</h1>
+            <p>
+              If you have an application you are interested in developing, a feature
+              that you need built or a project that needs coding. I'd love to help
+              with it
+            </p>
+            <form class="form" action="https://formspree.io/f/xlezygkr" method="POST">
```
Comment (1GORDON) [inline index.html]: Great job team 🤝

- [ ] [Optional] You may please add the dark line at the bottom of the design.
- Your design
your design

- Figma design
figma design

File: index.html [hunk 3]
```diff
@@ -1,138 +1,215 @@
...
 
-</body>
-</html>
\ No newline at end of file
+            <h1>CONTACT US</h1>
+            <p>
+              If you have an application you are interested in developing, a feature
+              that you need built or a project that needs coding. I'd love to help
+              with it
+            </p>
```
Comment (1GORDON) [inline index.html:176]: Great job team 🤝

- [ ] [Optional] You may try to make the font family of your text throughout the whole project a replica of the one provided in the Figma design.

File: styles.css [hunk 4]
```diff
@@ -203,3 +208,87 @@ h3 {
...
+  font-size: 20px;
+  line-height: 28px;
+  padding-bottom: 40px;
+}
+
+.btn:hover {
+  background: #7d73fc;
+  color: #fff;
+  font-size: 12px;
+}
```
Comment (1GORDON) [inline styles.css:281]: Great job team 🤝

- [ ] [Optional] You may as well add the `cursor: pointer` property to the buttons to add the UX experience.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi team ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it :ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages  ✔️
- Professional Pull Request ✔️
- Professional Readme ✔️
## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**